src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,57 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int MAX_VALUE = 11;
+    private static final int DEFAULT_INDEX = -1;
+    private static final int CAPACITY = 12;
 
-    public int total = -1;
+    private final int[] NUMBERS = new int[getCAPACITY()];
 
-    public void countIn(int in) {
+    public static int getMaxValue() {
+        return MAX_VALUE;
+    }
+
+    public static int getDefaultIndex() {
+        return DEFAULT_INDEX;
+    }
+
+    public static int getCAPACITY() {
+        return CAPACITY;
+    }
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = getDefaultIndex();
+
+    protected void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            getNUMBERS()[++total] = in;
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    protected boolean callCheck() {
+        return total == getDefaultIndex();
     }
 
-    public boolean isFull() {
-        return total == 11;
+    protected boolean isFull() {
+        return total == getMaxValue();
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
-        return NUMBERS[total];
+            return getDefaultIndex();
+        return getNUMBERS()[total];
     }
 
-    public int countOut() {
+    protected int countOut() {
         if (callCheck())
-            return -1;
-        return NUMBERS[total--];
+            return getDefaultIndex();
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)